icechunk-python/python/icechunk/__init__.py

```diff
@@ -71,6 +82,9 @@
     "VirtualChunkContainer",
     "__version__",
     "containers_credentials",
```

Comment:
you'll need to add the `gcs_storage` function in storage.py too

Reply:
done!